supplier_calendar/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020 ForgeFlow
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Supplier Calendar",
+    "summary": "Supplier Calendar",
+    'version': '12.0.1.0.0',
+    "category": "Purchase Management",
+    "author": "ForgeFlow",
+    "contributors": [
```

Comment:
did you meant ``maintainers`` : https://github.com/OCA/maintainer-tools/blob/master/template/module/__manifest__.py#L13